s4e-backend/src/main/java/pl/cyfronet/s4e/bean/Product.java

```diff
@@ -57,4 +64,23 @@
     @Type(type = "jsonb")
     @ToString.Exclude
     private Map<String, Object> searchableMetadata;
+
+    @ManyToMany(cascade = {
+            CascadeType.PERSIST,
+            CascadeType.MERGE
+    })
+    @JoinTable(name = "product_app_users",
+            joinColumns = @JoinColumn(name = "product_id"),
+            inverseJoinColumns = @JoinColumn(name = "app_user_id")
+    )
+    @ToString.Exclude
+    private Set<AppUser> favourites = new HashSet<>();
```

Comment:
The default value will be ignored by the builder, it passes tests without it.
If you want to keep the default use @Builder.Default on the field.